src/main/java/com/artipie/rpm/http/RpmUpload.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.http;
+
+import com.artipie.asto.Storage;
+import com.artipie.http.Slice;
+import com.artipie.http.rq.RqMethod;
+import com.artipie.http.rt.RtRule;
+import com.artipie.http.rt.SliceRoute;
+import com.artipie.http.slice.SliceUpload;
+
+/**
+ * Slice for rpm packages upload.
+ *
+ * @since 0.8.3
+ * @todo #162:30min Finish implementation of RpmUpload
+ *  RpmSlice should behave like the defined in
+ *  https://github.com/artipie/rpm-adapter/issues/162. Finish the
```

Comment:
@paulodamaso I'd me more concrete with this puzzle and say what exactly is missing now, e.g.:
 - parse `override` query param
 - trigger metadata update

Otherwise, you can just put it to integration tests and add the puzzle to fix tests and remove this puzzle